src/modules/shared/components/ArchivedBanner.tsx

```
@@ -6,21 +8,93 @@ import { Archive, CaretDown } from "@phosphor-icons/react";
   itemType,
-  onCtaClick,
+  setId,
+  songId,
 }) => {
   const [isExpanded, setIsExpanded] = useState<boolean>(true);
 
+  const { userId, isLoaded: isAuthLoaded } = useAuth();
+  const router = useRouter();
+
+  const { data: userData, error: userQueryError } = api.user.getUser.useQuery(
+    { userId: userId! },
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)